cfme/fixtures/chargeback_rate.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+
+from random import random, choice
+
+import pytest
+import fauxfactory
+
+from cfme.intelligence import chargeback as cb
+
+
+def new_chargeback_rate(appliance, interval, include_variable_rates=True):
+
+    # Create a new Chargeback compute rate
```

Comment:
1. For a test fixture, I don't think the variable rate should be random, it creates too much uncontrolled variability between test runs that isn't parametrized or immediately accessible as a variance between two tests runs with different results.

2. These fixtures shouldn't take passed arguments, only other fixtures, as they shouldn't be called like methods but only included as fixtures for a test.  This can be 'fixed' by creating separate fixtures, one including a variable cb rate and one not. The test can then parametrize on the two fixtures.

3. Below where the cb rate is actually created via copy/add depending on a random choice, again this shouldn't be random. It should just use add (potentially though rest even), this isn't a 'test' of cb rate creation.